From f58c88ea261670d4287b0d1df262812db206a347 Mon Sep 17 00:00:00 2001
From: Stefan Lellwitz <lel@iesy.com>
Date: Thu, 3 Feb 2022 11:45:53 +0100
Subject: [PATCH 1/1] arm64: Makefile: consider dtbs apart from Rockchip

We need to be able to select between Rockchip and iesy dtb

Signed-off-by: Stefan Lellwitz <lel@iesy.com>
---
 arch/arm64/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/Makefile b/arch/arm64/Makefile
index 5b3908880485..3f17484dc11e 100644
--- a/arch/arm64/Makefile
+++ b/arch/arm64/Makefile
@@ -199,7 +199,7 @@ endif
 LOGO := $(notdir $(wildcard $(srctree)/logo.bmp))
 LOGO_KERNEL := $(notdir $(wildcard $(srctree)/logo_kernel.bmp))
 
-%.img: rockchip/%.dtb kernel.img $(LOGO) $(LOGO_KERNEL)
+%.img: %.dtb kernel.img $(LOGO) $(LOGO_KERNEL)
 	$(Q)$(srctree)/scripts/mkimg --dtb $*.dtb
 
 CLEAN_DIRS += out
-- 
2.34.1

